pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>edu.wisc.library.ocfl</groupId>
       <artifactId>ocfl-java-core</artifactId>
-      <version>0.0.7-SNAPSHOT</version>
+      <version>0.1.0</version>
```

Comment:
@pwinckles : do you plan on following this PR with a `0.2.0-SNAPSHOT` update?

Reply:
I hadn't planned on it. That snapshot doesn't exist. I don't have automatic snapshot build setup for ocfl-java.

Reply:
I am happy to merge this PR, but it seems that we will need to bump to the next SNAPSHOT version before we make any more changes to fcrepo-storage-ocfl.

Reply:
Why is that?

Reply:
I was reading the PR as updating the version of ocfl-java-core (itself) to a non-snapshot version... we are all good. Thanks.